models/evc.py

```diff
@@ -867,9 +869,10 @@ def deploy_to_path(self, path=None):  # pylint: disable=too-many-branches
                 use_path = Path()
                 self._install_direct_uni_flows()
             else:
-                log.warning(
-                    f"{self} was not deployed. No available path was found."
-                )
+                msg = f"{self} was not deployed. No available path was found."
+                if tag_errors:
+                    msg = self.add_tag_errors(msg, tag_errors)
+                log.warning(msg)
```

Comment:
@Alopalao, can we have a `log.error` instead of `log.warning` when `tag_errors` isn't empty?

As a general rule of thumb, we've use warning for really just as a heads up, or for things that might recover like something transitory like a retry and so on. A path not found can still be just a warning since sometimes the path doesn't exist due to link not available, but when failing to allocate a tag that's an error. See what I mean? 

The good thing about keeping them consistent is that errors won't be overlooked as warnings.

Reply:
Got it, changed.